loader/loader.go

```diff
@@ -185,6 +185,7 @@ func Load(configDetails types.ConfigDetails, options ...func(*Options)) (*types.
 			LookupValue:     configDetails.LookupEnv,
 			TypeCastMapping: interpolateTypeCastMapping,
 		},
+		ResolvePaths: true,
```

Comment:
@ndeloof this is a change in default behavior right? Before this was false by default.
If I explicitly set it to false through the options argument of the `Load` function the behavior would be the same as before this change right?

Reply:
The 'ResolvePaths' make the unit-test fail in the nerdctl :-)
relate : https://github.com/containerd/nerdctl/pull/2338

Reply:
@tomfeigin code that has been moved into `ResolveRelativePaths` used to be inside `Normalize` which is enabled until you explicitly set `SkipNormalization`, so this is not a behavior change. Just those two processing aspects have been separated into distinct flags.